### PR TITLE
[ozone] Proper handeling of _get_scheme function in ozone

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- It was breaking HDFS upload

## How was this patch tested?

- Locally via uploading a file

### CODE REVIEW

File: desktop/core/src/desktop/lib/fs/ozone/upload.py [hunk 1]
```diff
@@ -120,9 +120,9 @@ def _is_ofs_upload(self):
 
   def _get_scheme(self):
     if self.destination:
-      dst_parse = lib_urlparse(self.destination)
```
Comment (Harshg999) [inline desktop/core/src/desktop/lib/fs/ozone/upload.py:123]: Need to Remove `lib_urlparse` imports.

Comment (agl29) [reply desktop/core/src/desktop/lib/fs/ozone/upload.py:123]: Done

**Review by Harshg999 — APPROVED** [review]
LGTM!

**Review by wing2fly — APPROVED** [review]
@agl29 @Harshg999  Add unit test for different file system!

**Comment by Harshg999** [comment]
@wing2fly - Added UT for checking different FS.

**Pull request merged**